### PR TITLE
Feat/advisory CVE 2023 1668

### PR DESCRIPTION
**Pull request by garloff**
@berendt: Please fill in betacloud, regio tech information and correct 4.3.0 status. Also: If Jens should receive some praise, please add.
@jnull, @linwalth, @chschilling: Please fill in status of WaveStack, which may include forward looking statements in case you can't patch today.

I want to push this out this afternoon.

### CODE REVIEW

File: _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+## Patch status of SCS clouds
+
+The networking improvements for OSISM 5.1.0 were done in close alignement between
+PlusServer and OSISM -- consequently, all regions of the 
+[PlusCloud Open](https://www.plusserver.com/en/products/pluscloud-open)
+had been upgraded to OSISM-5.1.0 in calendar week 15 already.
+
+[Betacloud](https://betacloud.de/) - is it currently operational?
+[WaveStack](https://www.noris.de/wavestack-cloud/) - still on OSISM 4.x as of Fri morning
+[regio tech](https://regio.digital/) - On 5.x, exact status to be confirmed
```
Comment (berendt) [inline _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md:80]: 5.1.0

File: _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+urgent need for it.
+
+## Patch status of SCS clouds
+
+The networking improvements for OSISM 5.1.0 were done in close alignement between
+PlusServer and OSISM -- consequently, all regions of the 
+[PlusCloud Open](https://www.plusserver.com/en/products/pluscloud-open)
+had been upgraded to OSISM-5.1.0 in calendar week 15 already.
+
+[Betacloud](https://betacloud.de/) - is it currently operational?
```
Comment (berendt) [inline _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md:78]: It's operational, yes. We run the rolling tag images there, not the stable releases.

File: _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+of CVE-2023-1668.
+
+SCS release 3 (OSISM 4.x.y) is only officially supported until the end of April
+2023 -- we advise all our partners to move to R5 with OSISM 5.1.0 or later as
+soon as possible.
+
+OSISM has worked on preparing a version 4.3.0 that includes a patched OvS version
+for those partners that can not yet move to R4/OSISM-5.x. as of today (2023-04-21),
+it has not been released yet -- please get in touch with OSISM if you have an
+urgent need for it.
```
Comment (berendt) [inline _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md:69]: Release planned for end of April.

Comment (garloff) [reply _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md]: OK, thanks for the input!
I included it in the advisory.

**Comment by garloff** [comment]
So, from my perspective, this is ready for publication.
Please approve and merge if you agree.
Please speak up if you disagree.
If noone says anything, I'll push it out myself tonight, when I'm back at the computer.

**Review by fkr — APPROVED** [review]
LGTM flying over it. 
I will not have the time to read this in detail until later (2200 CEST) tonight.
If it is not merged then, I will read it in detail.

File: _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  - "kgarloff.jpg"
+  - "fkr.jpg"
+about:
+  - "garloff"
+  - "fkr"
+---
+
+## The vulnerability
+
+David Marchand (RedHat) reported a an issue with Open vSwitch (OvS) where
```
Comment (fkr) [inline _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md:20]:
```suggestion
David Marchand (RedHat) reported an issue with Open vSwitch (OvS) where
```

File: _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+## Impact on the SCS reference implementation
+
+Open vSwitch is used in almost all OpenStack setups -- specifically all SCS
+setups known to us use OvS. This includes both the OVN configurations
+(the SCS default) as well as those that rely on OvS alone.
+
+## Mitigation
+
+It is possible to set flow rules that block IP protocol 0 packets by adding
+3 flow rules as highest priority rules to ever OvS bridge. Care is needed to
```
Comment (fkr) [inline _i18n/en/_posts/blog/2023-04-21-cve-2023-1668.md:41]:
```suggestion
3 flow rules as highest priority rules to every OvS bridge. Care is needed to
```

**Comment by fkr** [comment]
Read over it now and fixed couple typos. From my side good to go.

**Comment by garloff** [comment]
Thanks!

**Pull request merged**